target/scripts/startup/setup-stack.sh

```diff
@@ -437,10 +437,17 @@ function _setup_ldap
 
   declare -A _dovecot_ldap_mapping
```

Comment:
We use uppercase variables, see CONTRIBUTING.md for styling

Reply:
Are underscore prefixes for internal variables still preferred, or should this just be DOVECOT_LDAP_MAPPING?

Reply:
It should just be `DOVECOT_LDAP_MAPPING`. Underscores are for function :)